primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:10:54.83,0:10:56.40,Default,,0,0,0,,Follow your orders!
 Dialogue: 0,0:10:57.60,0:10:59.84,Default,,0,0,0,,Analyzing.
 Dialogue: 0,0:10:59.84,0:11:01.57,Default,,0,0,0,,Core detected in center of object.
-Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's exterior is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
+Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's carapace is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
 Dialogue: 0,0:11:08.78,0:11:09.73,Default,,0,0,0,,Are you listening, Rin?
-Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,There's no time for emotions.
+Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,Don't let your emotions get the best of you.
```

Comment:
This doesn't seem harsh enough. I'd revert.
```suggestion
Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,There's no time for emotions.
```

Reply:
Seeing the two side-by-side, I agree, but "There's no time for emotions." is pretty awkward.
Settling on "There's no time to get all emotional."